frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/SlideInBackgroundable.java

```diff
@@ -76,15 +76,7 @@ public void run(ProgressIndicator indicator) {
         targetBranchLayout = branchLayout;
       } else {
         entryToSlideIn = entryByName.withChildren(List.empty());
-        try {
-          targetBranchLayout = branchLayout.slideOut(slideInOptions.getName());
-        } catch (EntryDoesNotExistException e) {
```

Comment:
Thing to make sure now: whether sliding in a managed branch under a new parent still works :goncern:

Reply:
> whether it fails when the provided branch name is invalid
what do you mean?

Reply:
Ah ok, nvm 🤦🏻